source/MaterialXContrib/MaterialXNode/MaterialXShadingNodeImpl.cpp

```diff
@@ -239,6 +272,15 @@ void MaterialXShadingNodeImpl<BASE>::updateShader(MHWRender::MShaderInstance& sh
         envRadiancePath, envIrradiancePath);
 
     MaterialX::DocumentPtr document = materialXData->getDocument();
+
+    // Look for any udimset on the document to use for texture binding.
+    mx::ValuePtr udimSetValue = document->getGeomAttrValue("udimset");
+    const mx::StringVec* uvdimStrings = nullptr;
+    if (udimSetValue && udimSetValue->isA<mx::StringVec>())
+    {
+        uvdimStrings = &(udimSetValue->asA<mx::StringVec>());
```

Comment:
We are given an explicit list of udims to use so don't need the overkill of the Maya python scripts which tries to scan all files on disk to find the udims to use.